src/lib/badge/_badge-theme.scss

```diff
@@ -0,0 +1,177 @@
+@import '../core/theming/palette';
+@import '../core/theming/theming';
+@import '../core/typography/typography-utils';
+
+$mat-badge-font-size: 12px;
+$mat-badge-font-weight: 600;
+$mat-badge-size: 22px !default;
+
+.mat-badge {
+  position: relative;
+}
+
+.mat-badge:after,
```

Comment:
These `:after` instances should be `::after`. I think we had a linting rule for it at some point.